### PR TITLE
[AAE-6295] Updated attach-file-widget-cloud.page.ts

### PR DESCRIPTION
**Pull request by Sushmitha-Vk**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-6295

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by sivakumar414ram — APPROVED** [review]

File: lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts [hunk 1]
```diff
@@ -63,6 +63,12 @@ export class AttachFileWidgetCloudPage {
         }
     }
 
+    async checkNoFileIsAttached(): Promise<void> {
```
Comment (arditdomi) [inline lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: Are you sure we need this? Why a new method? My impression is we need to update this page https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L44-L48
and also the ADW testing pages with the new locators

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: its required in APA, to update failing tests

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: there is a condition to check for no files, so this method is required

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: the cloud attach-widget page was updated while merging attach-file-widget changes only. Only I have to update the ADW APA which I'm creating a new PR in apps, and this function was missing from attach-widget-cloud page so added it

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: If there are failing tests in APA due to a change, it is expected to update an existing method. Why are we creating a new one? The method that you are creating exists in this page https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L44-L48

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: And this one as well, is being used in a failing test from APA. We need to update the locator https://github.com/Alfresco/alfresco-ng2-components/blob/d3863077cf0b18d6f210f987f27d13fe3358aa3e/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L168

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: Yes, but we are not using `AttachFileWidgetPage` for APA but using `AttachFileWidgetCloudPage` which requires this method

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: As only the design of `attach-file-cloud-widget.component` is changed mat-list remains the same in  `attach-file-widget.component` needed to update the attach-file cloud widget page

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: Are you sure? We are using it here https://github.com/Alfresco/alfresco-apps/blob/develop/e2es/content-ee-apa/process-services-cloud-extension-e2e/src/tests/start-process/start-process-on-personal-files.e2e.ts#L133-L135
and in many other places. All the e2e related to processes are using it

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: Yes, I have updated these files in the new PR that I'm going to create now in apps

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: the methods they are using are from ADF, shouldn't you be updating the ADF methods? Are you creating new methods in apps?

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: I haven't created any methods in apps, I have already updated the methods in the last ADF PR https://github.com/Alfresco/alfresco-ng2-components/pull/7324/files where I made changes for attach-file-widget, so only e2es passed for me when I merged new features of attach-file-widget. Just this method was missing in `AttachFileWidgetCloudPage` which is required in apps to check for no files.

Comment (mauriziovitale) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: Guys, do we need a chat?

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: You haven't updated all of them, I still see mat-list-item locators https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L73

Comment (arditdomi) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: yes let's continue this on a call

Comment (Sushmitha-Vk) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:66]: yes, mat-list-item still exists for` attach-file-widget` component, only `attach-file-widget-cloud` component ui is changed

**Review by arditdomi — APPROVED** [review]

File: lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts [hunk 2]
```diff
@@ -63,6 +63,12 @@ export class AttachFileWidgetCloudPage {
...
+        const filesListLocator = 'div[class="adf-file-properties-table"]';
+        const fileItem = this.widget.$(filesListLocator).$('table');
```
Comment (eromano) [inline lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:68]: can we use the TestElement from testing core lib? lib/testing/src/lib/protractor/core/test-element.ts

**Review by eromano — CHANGES_REQUESTED** [review]

File: lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts (hunk 2 above)
Comment (eromano) [reply lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts:68]: evrywhere

**Review by eromano — APPROVED** [review]

**Pull request merged**